package.json

```diff
@@ -28,7 +28,7 @@
     "pluralize": "^8.0.0"
   },
   "peerDependencies": {
-    "@prisma/client": "^2.0.0-preview022",
```

Comment:
I think we should bump the `devDependencies` one too, so that we're testing with what our contract with users is.